### ISSUE TITLE
CorsService exception on October CMS 2

### ISSUE THREAD

**Issue by marinne** [OFFLINE-GmbH/oc-cors-plugin#11]
Description:

Offline.Cors plugin throws an exception:

`in_array() expects parameter 2 to be array, null given`

on

`~/plugins/offline/cors/classes/CorsService.php line 33`

Environment:

- October CMS 2.1.17
- Offline.CORS 1.0.6
- PHP 7.4.22 (also on PHP 7.4.4)

How to reproduce:

1. Install the plugin.
2. On the backend > Settings > CORS-Settings page, add an entry to the Allowed Origins repeater and save.
3. The above exception occurs.

Additional info:

I tried to print the content of the $options array (CorsService.php line#23) and got the following:

```
[2021-10-27 02:44:49] local.DEBUG: array (
  'supportsCredentials' => false,
  'allowedOrigins' => NULL,
  'allowedHeaders' => NULL,
  'allowedMethods' => NULL,
  'exposedHeaders' => NULL,
  'maxAge' => 0,
) 
```
Looks like the `allowed*` entries should be an empty array but became NULL instead for some reason.

**Comment by huynhnamegal** [comment]
Me too, I used curl --head command for the API link and its didnot return any header.

**Comment by tobias-kuendig** [comment]
Fixed in release 1.0.7, thank you for bringing this up!
